src/services/article.tsx

```diff
@@ -30,3 +30,17 @@ export const getArticles = async (
 
 export const getNews = async (): Promise<AxiosResponse<IPage<IArticle[]>>> =>
   apiClient.get(`/articles/news`);
+
+export const getPopularReviews = async () =>
+  apiClient({
+    method: 'GET',
+    url: '/articles/reviews/comment-num',
+  });
```

Comment:
```suggestion
  apiClient.get('/articles/reviews/comment-num');
```

要统一啊  你如果这样写 那你就把上面所有的改成你这样